Src/VsVimShared/VsVimHost.cs

```diff
@@ -960,9 +960,16 @@ private FSharpOption<ListItem> NavigateToLocation(NavigationKind navigationKind,
             return null;
         }
 
-        public override void Make(bool jumpToFirstError, string arguments)
+        public override void Make(bool buildSolution, string arguments)
         {
-            SafeExecuteCommand(null, "Build.BuildSolution");
+            if (buildSolution)
+            {
+                SafeExecuteCommand(null, "Build.BuildSolution");
+            }
+            else
+            {
+                SafeExecuteCommand(null, "Build.BuildOnlyProject");

```

Comment:
Why `Build.BuildOnlyProject` instead of `Build.BuildSelection`? The latter is generally what I use for "current project".

Reply:
`BuildSelection` builds a project currently selected in the solution explorer. It does not necessarily have to be related to the file being edited currently. `BuildOnlyProject` builds the *current project* based on the buffer being edited.

IMHO, it might work for you because VS actively changes the selected project in the **SolutionExplorer** to match the current project. 

Note that it is also possible to select multiple projects in the solution explorer and call `BuildSelection` which is not something I'd want `:make` to do, given that vim is buffer-centric.

Reply:
Thanks. I wasn't aware of this command before this.